tests.md

```diff
@@ -0,0 +1,174 @@
+# Testing of Terraform Modules Explained
```

Comment:
Do you have any thoughts on how to monitor the percentage of test coverage metric?

Reply:
Ah no, not really.